src/Components/TodosFooter.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import classNames from 'classnames';
+import { ListAction } from '../Enum/ListAction';
+import { useTodo } from '../Hooks/UseTodo';
+
+export const TodosFooter: React.FC = () => {
+  const {
+    todos,
+    setTodos,
+    filter,
+    setFilter,
+  } = useTodo();
+  const selectedTodos = todos.filter(todo => !todo.completed);
+  const compleatedTodosLength = todos.filter(todo => todo.completed).length;
+
+  const deleteTodos = () => {
+    setTodos(selectedTodos);
+  };
```

Comment:
clear completed button doesn't work correctly
you only remove todos on state, but not on server. after reload page completed todos stilll exists.
and should be visible loader for each todo which in delete process.